### DISCUSSION TITLE
Mid360 with ROS | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1747] [Q&A]
# Mid360 with ROS | Seeed Studio Wiki

This wiki provides a step-by-step guide to install and set up MID360 LiDAR on reComputer J30/40 using ROS.

https://wiki.seeedstudio.com/mid360/

**Comment by neduser** [comment]
This code is not work for me. 
When I copy
git clone https://github.com/Livox-SDK/Livox-SDK2.git
cd ./Livox-SDK2/

I  see that 
Comment git not found, but can be installed with: 
sudo apt install git

Second he write me:
Unable to locate package clone
Unable to locate package https://github.com/Livox-SDK/Livox-SDK2.git

Can you help me. What I wrong to do?

**Reply by neduser** [reply]
I try to do it second time, copy command and paste in terminal


git clone https://github.com/Livox-SDK/Livox-SDK2.git
cd ./Livox-SDK2/
mkdir build
cd build
cmake .. && make -j
sudo make install
Command 'git' not found, but can be installed with:
sudo apt install git
bash: cd: ./Livox-SDK2/: No such file or directory
CMake Warning:
  Ignoring extra path from command line:

   ".."


CMake Error: The source directory "/home/neduser" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
make: *** No rule to make target 'install'.  Stop.
neduser@neduser-PC:~/build$ sudo apt install git clone https://github.com/Livox-SDK/Livox-SDK2.git
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package clone
E: Unable to locate package https://github.com/Livox-SDK
neduser@neduser-PC:~/build$
